packages/build/webpack/constants.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+const lkModule = process.env.LK_MODULE;
+const lkModuleContainer = process.env.LK_MODULE_CONTAINER;
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+// This path assumes the enlistment in labkey-ui-components is a sibling of the root of the LabKey enlistment.
+// TODO figure out how we can make this more flexible and get an absolute path from the dev based on their setup
+const labkeyUIComponentsRelPath = (lkModuleContainer ? "../../../../../" : "../../../../") + "labkey-ui-components/packages/components";
+const labkeyUIComponentsPath = process.env.LINK ? path.resolve(labkeyUIComponentsRelPath) : path.resolve("./node_modules/@labkey/components");
+console.log("Using @labkey/components path: " + labkeyUIComponentsPath);
+
+const freezerManagerRelPath = (lkModuleContainer ? "../../../../../../" : "../../../../../") + "inventory/packages/freezermanager";
+const freezerManagerPath = process.env.LINK ? path.resolve(__dirname, freezerManagerRelPath) : path.resolve("./node_modules/@labkey/freezermanager");
+console.log("Using @labkey/freezermanager path: " + freezerManagerPath);
```

Comment:
It would be nice if we could hide this behind an environment variable as well.  Or just add it locally for the modules that need it.  Seems like majority of builds will not need this and it's a little weird seeing output about freezer manager in all webpack build logs.  Also just seems a little specific for a shared config.

All these config files are just big JS objects, wonder if we could add this and the alias in watch.config.js at the local level?

Reply:
I agree. I added a TODO to look into this for the next round of changes.